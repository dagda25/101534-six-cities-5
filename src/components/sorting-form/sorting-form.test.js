import React from "react";
import renderer from "react-test-renderer";
import SortingForm from "./sorting-form";


describe(`Render SortingForm`, () => {
  it(`Render SortingForm`, () => {
    const tree = renderer
      .create(
          <SortingForm
            changeSorting={() => {}}
            currentSorting={`Popular`}
            isMenuOpen={false}
            onToggleClick={() => {}}
          />,
          {
            createNodeMock: () => {
              return {};
            }
          }
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});

