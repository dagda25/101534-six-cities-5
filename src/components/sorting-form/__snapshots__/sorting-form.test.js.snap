// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render SortingForm Render SortingForm 1`] = `
<form
  action="#"
  className="places__sorting"
  method="get"
>
  <span
    className="places__sorting-caption"
  >
    Sort by
  </span>
  <span
    className="places__sorting-type"
    data-type="toggler"
    onClick={[Function]}
    tabIndex="0"
  >
    Popular
    <svg
      className="places__sorting-arrow"
      height="4"
      width="7"
    >
      <use
        xlinkHref="#icon-arrow-select"
      />
    </svg>
  </span>
  <ul
    className="places__options places__options--custom"
    data-value="popular"
    onClick={[Function]}
  >
    <li
      className="places__option places__option--active"
      onClick={[Function]}
      tabIndex="0"
    >
      Popular
    </li>
    <li
      className="places__option"
      onClick={[Function]}
      tabIndex="0"
    >
      Price: low to high
    </li>
    <li
      className="places__option"
      onClick={[Function]}
      tabIndex="0"
    >
      Price: high to low
    </li>
    <li
      className="places__option"
      onClick={[Function]}
      tabIndex="0"
    >
      Top rated first
    </li>
  </ul>
  <select
    className="places__sorting-type"
    id="places-sorting"
    onChange={[Function]}
    style={
      Object {
        "display": "none",
      }
    }
    value="Popular"
  >
    <option
      className="places__option"
      value="Popular"
    >
      Popular
    </option>
    <option
      className="places__option"
      value="Price: low to high"
    >
      Price: low to high
    </option>
    <option
      className="places__option"
      value="Price: high to low"
    >
      Price: high to low
    </option>
    <option
      className="places__option"
      value="Top rated first"
    >
      Top rated first
    </option>
  </select>
</form>
`;
